Homeworks/Homework W8/SQLQuery_HomeworkW8.sql

```diff
@@ -0,0 +1,340 @@
+CREATE DATABASE ZooManagemnt;
+
+USE ZooManagemnt;
+-- Creating tables
+CREATE TABLE ZoneArea
+(
+Id INT IDENTITY(1,1) PRIMARY KEY,
+Zone VARCHAR(50)
+);
+
+CREATE TABLE Species
+(
+Id INT IDENTITY(1,1) PRIMARY KEY,
+Name VARCHAR(50),
+Zone INT REFERENCES ZoneArea(Id)
+);
+
+CREATE TABLE Animals
+(
+Id INT IDENTITY(1,1) PRIMARY KEY,
+Species INT REFERENCES Species(Id)
```

Comment:
SpieceId (explanation in a comment below). Also pluralization should not be used usually for column names - it suggests that insider that column a list of values is stored which would be wrong.